app/src/main/java/ch/epfl/sdp/drone3d/service/impl/mission/ParallelogramMappingMissionService.kt

```diff
@@ -9,13 +9,15 @@ import ch.epfl.sdp.drone3d.model.mission.*
 import ch.epfl.sdp.drone3d.service.api.drone.DroneService
 import ch.epfl.sdp.drone3d.service.api.mission.MappingMissionService
 import com.mapbox.mapboxsdk.geometry.LatLng
-import java.lang.IllegalArgumentException
 import javax.inject.Inject
+import kotlin.math.PI
 
 class ParallelogramMappingMissionService @Inject constructor(val droneService: DroneService): MappingMissionService {
```

Comment:
Some documentation would be welcome here

Reply:
You are right, I am on it